### PR TITLE
Add a Buildkite pipeline

### PR DESCRIPTION
**Pull request by ven-k**

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
We should also have CPU linux

**Comment by ChrisRackauckas** [comment]
Windows isn't necessary

**Comment by foldfelis** [comment]
Thanks, looks good to me

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -21,7 +21,6 @@ jobs:
           - '1'
           # - 'nightly'
         os:
-          - ubuntu-latest
           - windows-latest
```
Comment (foldfelis) [inline .github/workflows/CI.yml]: To enable GitHub CI on linux for CPU

```suggestion
          - ubuntu-latest
          - windows-latest
```

**Pull request merged**